File: src/main/java/com/zippeykeys/praisebe/registry/MultiblockRegistry.java
package com.zippeykeys.praisebe.registry;

import com.zippeykeys.praisebe.Reference;
import com.zippeykeys.praisebe.block.multi.Multiblock;

import lombok.NoArgsConstructor;
import net.minecraftforge.fml.common.Mod.EventBusSubscriber;

@NoArgsConstructor
@EventBusSubscriber(modid = Reference.MOD_ID)
public class MultiblockRegistry extends PBRegistry<Multiblock> {

}

File: src/main/kotlin/com/zippeykeys/praisebe/registry/MultiblockRegistry.kt
package com.zippeykeys.praisebe.registry

import com.zippeykeys.praisebe.Reference
import com.zippeykeys.praisebe.block.multi.Multiblock
import net.minecraftforge.fml.common.Mod.EventBusSubscriber

@EventBusSubscriber(modid = Reference.MOD_ID)
class MultiblockRegistry : PBRegistry<Multiblock>()